### PR TITLE
Dashboard deploy

### PR DESCRIPTION
**Pull request by joshenlim**

### CODE REVIEW

File: packages/ui/src/components/Command/Command.utils.tsx [hunk 1]
```diff
@@ -312,28 +312,24 @@ CommandShortcut.displayName = 'CommandLabel'
...
-    return text.replace(
-      regex,
-      (match) => `<span class="font-semibold text-scale-1200">${match}</span>`
-    )
-  }
+    return returnValue
+  })
 
-  return <span dangerouslySetInnerHTML={{ __html: highlightMatches(text) }} {...props} />
+  return <span {...props}>{elements}</span>
```
Comment (MildTomato) [inline packages/ui/src/components/Command/Command.utils.tsx:332]: this looks an odd change. someone QA'ed this ?

Comment (MildTomato) [reply packages/ui/src/components/Command/Command.utils.tsx:332]: Looks ok to me actually

File: packages/ui/src/components/Command/DocsSearch.tsx [hunk 2]
```diff
@@ -185,9 +185,11 @@ const DocsSearch = () => {
                     <CommandLabel>
                       <TextHighlighter text={page.meta.title} query={search} />
                     </CommandLabel>
-                    <div className="text-xs text-scale-900">
-                      <TextHighlighter text={page.meta.description} query={search} />
-                    </div>
+                    {page.meta.description && (
+                      <div className="text-xs text-scale-900">
```
Comment (MildTomato) [inline packages/ui/src/components/Command/DocsSearch.tsx:189]: `text-light`

File: studio/components/interfaces/Auth/Users/UsersList.tsx [hunk 3]
```diff
@@ -45,16 +61,17 @@ const UsersList = ({}) => {
                   <div className="flex items-center space-x-3 opacity-75">
                     <IconAlertCircle size={16} strokeWidth={2} />
                     <p className="text-scale-1000">
```
Comment (MildTomato) [inline studio/components/interfaces/Auth/Users/UsersList.tsx:63]: `text`

File: studio/components/interfaces/Auth/Users/UsersListItem.tsx [hunk 4]
```diff
@@ -29,9 +28,7 @@ const UserListItem = ({ user, canRemoveUser, canRemoveMFAFactors }: UserListItem
         <span className="text-scale-1200">{!user.phone ? '-' : user.phone}</span>
       </Table.td>
       <Table.td className="table-cell">
-        <span className="capitalize text-scale-1200">
-          {user?.raw_app_meta_data?.provider || user?.app_metadata?.provider}
-        </span>
+        <span className="capitalize text-scale-1200">{user?.raw_app_meta_data?.provider}</span>
```
Comment (MildTomato) [inline studio/components/interfaces/Auth/Users/UsersListItem.tsx:31]: use `text` not `text-scale-1200`

File: studio/components/interfaces/Auth/Users/UsersPagination.tsx [hunk 5]
```diff
@@ -1,41 +1,49 @@
...
+  const hasPrevious = page > 1
+  const hasNext = toRow < total
 
   return (
     <nav className="flex items-center justify-between overflow-hidden" aria-label="Pagination">
       <div className="hidden sm:block">
         <p className="text-xs text-scale-900">
           Showing
-          <span className="px-1 font-medium text-scale-1100">{PageState.fromRow}</span>
+          <span className="px-1 font-medium text-scale-1100">{fromRow}</span>
```
Comment (MildTomato) [inline studio/components/interfaces/Auth/Users/UsersPagination.tsx:26]: `text-light`

**Review by MildTomato — APPROVED** [review]

**Comment by joshenlim** [comment]
will fix all the text stuff in another PR

**Pull request merged**